products/jbrowse-desktop/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "jbrowse-desktop",
-  "productName": "JBrowse 2",
+  "name": "@gmod/jbrowse-desktop",
+  "productName": "jbrowse-desktop",
```

Comment:
I can't remember what this field is used for, but it doesn't seem to change the executable name. I think you also have to change it in `build.productName` and `build.linux.executableName`.

Reply:
I proposed maybe removing this if it is unused?